ansible/roles/debops.postfix/templates/ldap/postfix__virtual_ldap_header.j2

```diff
@@ -0,0 +1,21 @@
+# {{ ansible_managed }}
+
+# Connection information for the LDAP directory
+
+server_host = {{ postfix__ldap_uri }}
+start_tls = {{ "yes" if (postfix__ldap_start_tls|bool) else "no" }}
+version = 3
+{% if postfix__pki|bool %}
+# File containing the X509 Certification Authority certificates in
+# PEM format which are to be recognized by the client  in  SSL/TLS connections
+##tls_ca_cert_file = {{ postfix__pki_ca_crt }}
+# Whether or not to request server's X509  certificate  and  check
+# its  validity  when  establishing SSL/TLS connections.
+# FIXME: set to yes again
+##tls_require_cert = no
+{% endif %}
+
+bind = yes
+bind_dn = {{ postfix__ldap_binddn }}
+bind_pw = {{ postfix__ldap_bindpw }}
```

Comment:
You will probably want to use this header file as an argument to manage the lookup tables directly instead of going through the `postfix__lookup_tables` variable. Actually I think that this might be better utilized by implementing `item.connection_config` parameter in the `debops.postfix` role itself so that it can be shared across many lookup tables easily. I'll look into it.